### ISSUE TITLE
Length of Participation [years] 

### ISSUE THREAD

**Issue by markus2330** [sirblackheart/elektrasurvey#73]
validator to only allow empty or integers?

**Comment by sirblackheart** [comment]
this seems to be already implemented?

**Comment by Namoshek** [comment]
I don't think we can add validation for single fields (or column) of a grid; at least I didn't find any option for that. We could probably script it ourselfs, but remains the question if it makes much sense if the validation is just browser side.

**Comment by sirblackheart** [comment]
I did try some approaches but none had valid results.

**Comment by markus2330** [comment]
We can also post-process it.

**Comment by Namoshek** [comment]
Probably the easiest solution...
